lib/signature.c

```diff
@@ -166,7 +167,7 @@ rpmRC rpmGenerateSignature(char *SHA256, char *SHA1, uint8_t *MD5,
 	 * Put the 64bit size variants into the header, but
 	 * modify spaceSize so that the resulting header has
 	 * the same size. Note that this only works if all tags
-	 * with a lower number than RPMSIGTAG_RESERVEDSPACE are
+	 * with a lower number than RPMSIGTAG_RESERVEDSPACE/PAD are
```

Comment:
This bogus comment about PAD gets nicely highlighted here. Fixed in 25ae44f475eeaa89a2c0b3e9dbac6a9f65a8b3a0 now.